src/app/fyle/my-view-report/my-view-report.page.ts

```
@@ -433,38 +435,46 @@ export class MyViewReportPage {
-      });
-      this.trackingService.showToastMessage({ ToastContent: message });
-    });
-  }
-
-  submitReport(): void {
-    this.spenderReportsService.submit(this.reportId).subscribe({
-      next: () => {
+    this.submitReportLoader = true;
+    this.spenderReportsService
+      .resubmit(this.reportId)
+      .pipe(finalize(() => (this.submitReportLoader = false)))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)